### ISSUE TITLE
About box text is white

### ISSUE THREAD

**Issue by danielpunkass** [chipotle/tutti-colori-theme#1]
I'm not sure if this is a theme problem or an app problem, but I have only reproduced it with Tutti Colori so I thought I would file it here.

When Tutti Colori window theme is active, the About Box defaults to white text making it unreadable. If you switch to Tutti Color while the about box is open it's fine, but if you quit and reopen the app, the About Box defaults to the white text.

<img width="640" alt="Screen Shot 2020-05-16 at 4 34 04 PM" src="https://user-images.githubusercontent.com/14606/82129915-8aed8f80-9794-11ea-9710-ebe027ba21c3.png">

**Comment by chipotle** [comment]
That's an interesting one. :) I can duplicate it when I *start* Nova with the Tutti Colori window theme selected, but if I switch window themes and then switch back, the text is black.

If I switch the "vibrancy" setting of the theme from "none" to "vibrant", then the problem is solved, sort of: the about box becomes white text on dark. Until you switch window themes and then switch back, when it becomes dark text on white again.

In short: ¯\\\_(ツ)\_/¯ I am going to ask Panic if this is me or them. :)

**Comment by danielpunkass** [comment]
Cool, asking Panic sounds like a good idea! I noticed that when you choose Panic's default bright theme, the about box is dark mode after relaunching. Maybe that defaulting to dark mode behavior is somehow at fault, or involved at least.

**Comment by chipotle** [comment]
This *looks* like it was actually a problem with Nova, and has been fixed in the newest beta release (b13).

**Comment by danielpunkass** [comment]
Woot!
